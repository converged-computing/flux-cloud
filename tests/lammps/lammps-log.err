ðŸ§ª Prepared 1 experiment matrices
--2023-01-14 12:21:50--  https://raw.githubusercontent.com/flux-framework/flux-operator/main/examples/dist/flux-operator.yaml
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 2606:50c0:8003::154, 2606:50c0:8000::154, 2606:50c0:8001::154, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8003::154|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 22252 (22K) [text/plain]
Saving to: â€˜/tmp/flux-operator.213KOC.yamlâ€™

     0K .......... .......... .                               100%  112M=0s

2023-01-14 12:21:50 (112 MB/s) - â€˜/tmp/flux-operator.213KOC.yamlâ€™ saved [22252/22252]

Issue with run: run_command() got an unexpected keyword argument 'capture'
