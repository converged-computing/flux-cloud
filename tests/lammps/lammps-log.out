[0;35m   cluster  : flux-cluster[0m
[0;35m    version : 1.23[0m
[0;35m    size   : 4[0m
[0;35mrepository  : flux-framework/flux-operator[0m
[0;35m     branch : main[0m
minikube is installed
wget is installed
* Profile "minikube" not found. Run "minikube profile list" to view all profiles.
  To start a cluster, run: "minikube start"

[0;32mminikube start --nodes=4[0m
* minikube v1.26.1 on Ubuntu 22.04
* Automatically selected the docker driver
* Using Docker driver with root privileges
* Starting control plane node minikube in cluster minikube
* Pulling base image ...
* Creating docker container (CPUs=2, Memory=2200MB) ...
* Preparing Kubernetes v1.24.3 on Docker 20.10.17 ...
  - Generating certificates and keys ...
  - Booting up control plane ...
  - Configuring RBAC rules ...
* Configuring CNI (Container Networking Interface) ...
* Verifying Kubernetes components...
  - Using image gcr.io/k8s-minikube/storage-provisioner:v5
* Enabled addons: storage-provisioner, default-storageclass

* Starting worker node minikube-m02 in cluster minikube
* Pulling base image ...
* Creating docker container (CPUs=2, Memory=2200MB) ...
* Found network options:
  - NO_PROXY=192.168.49.2
* Preparing Kubernetes v1.24.3 on Docker 20.10.17 ...
  - env NO_PROXY=192.168.49.2
* Verifying Kubernetes components...

* Starting worker node minikube-m03 in cluster minikube
* Pulling base image ...
* Creating docker container (CPUs=2, Memory=2200MB) ...
* Found network options:
  - NO_PROXY=192.168.49.2,192.168.49.3
* Preparing Kubernetes v1.24.3 on Docker 20.10.17 ...
  - env NO_PROXY=192.168.49.2
  - env NO_PROXY=192.168.49.2,192.168.49.3
* Verifying Kubernetes components...

* Starting worker node minikube-m04 in cluster minikube
* Pulling base image ...
* Creating docker container (CPUs=2, Memory=2200MB) ...
* Found network options:
  - NO_PROXY=192.168.49.2,192.168.49.3,192.168.49.4
* Preparing Kubernetes v1.24.3 on Docker 20.10.17 ...
  - env NO_PROXY=192.168.49.2
  - env NO_PROXY=192.168.49.2,192.168.49.3
  - env NO_PROXY=192.168.49.2,192.168.49.3,192.168.49.4
* Verifying Kubernetes components...
* Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default

[0;32mkubectl get nodes[0m
NAME           STATUS     ROLES           AGE   VERSION
minikube       Ready      control-plane   68s   v1.24.3
minikube-m02   Ready      <none>          46s   v1.24.3
minikube-m03   Ready      <none>          29s   v1.24.3
minikube-m04   NotReady   <none>          1s    v1.24.3

[0;32mwget -O /tmp/flux-operator.213KOC.yaml https://raw.githubusercontent.com/flux-framework/flux-operator/main/examples/dist/flux-operator.yaml[0m
namespace/operator-system created
customresourcedefinition.apiextensions.k8s.io/miniclusters.flux-framework.org created
serviceaccount/operator-controller-manager created
role.rbac.authorization.k8s.io/operator-leader-election-role created
clusterrole.rbac.authorization.k8s.io/operator-manager-role created
clusterrole.rbac.authorization.k8s.io/operator-metrics-reader created
clusterrole.rbac.authorization.k8s.io/operator-proxy-role created
rolebinding.rbac.authorization.k8s.io/operator-leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/operator-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/operator-proxy-rolebinding created
configmap/operator-manager-config created
service/operator-controller-manager-metrics-service created
deployment.apps/operator-controller-manager created

[0;32mkubectl get namespace[0m
NAME              STATUS   AGE
default           Active   67s
kube-node-lease   Active   69s
kube-public       Active   69s
kube-system       Active   69s
operator-system   Active   0s

[0;32mkubectl describe namespace operator-system[0m
Name:         operator-system
Labels:       control-plane=controller-manager
              kubernetes.io/metadata.name=operator-system
Annotations:  <none>
Status:       Active

No resource quota.

No LimitRange resource.
